docs/api/datasets/non_geo_datasets.csv

```diff
@@ -28,6 +28,7 @@ Dataset,Task,Source,License,# Samples,# Classes,Size (px),Resolution (m),Bands
 `LoveDA`_,S,Google Earth,"CC-BY-NC-SA-4.0","5,987",7,"1,024x1,024",0.3,RGB
 `MapInWild`_,S,"Sentinel-1/2, ESA WorldCover, NOAA VIIRS DNB","CC-BY-4.0",1018,1,1920x1920,10--463.83,"SAR, MSI, 2020_Map, avg_rad"
 `Million-AID`_,C,Google Earth,-,1M,51--73,,0.5--153,RGB
+`MMEarth`_,"C, S","Aster, Sentinel, ERA5","CC-BY-4.0","100K--1M",,"128x128 or 64x64","multiple modalities",MSI
```

Comment:
Can we list out the resolutions instead of just saying "multiple modalities"?

Reply:
The paper on page 6 states ". Additionally, all pixel-level modalities were
re-projected to the Sentinel-2 10m grid, if needed, yielding a harmonized data
cube." So I guess actually everything is 10m in this dataset.

Reply:
Maybe @vishalned can comment (FYI, we are adding a data loader for MMEarth to TorchGeo!)

Reply:
Hey, yes exactly. every modality has been reprojected to 10m resolution. 

Also, just reading from the previous comments it seems like its a bit tricky with the nan values and the S1 bands (sorry about this). Incase the hdf5 is not useful to detect empty bands, i believe i also store some meta information regarding the bands (which are present and not) in the tile_info.json. 

Hope this helps, and let me know if you need any more clarifications